src/install.ts

```diff
@@ -55,7 +55,7 @@ export function install(Vue: VueConstructor) {
   }
 
   if (__DEV__) {
-    if (!Vue.version.startsWith('2.')) {
+    if (Vue.version[0] !== '2') {
```

Comment:
It's probably ok as we would not expect to see Vue 20 or something, but as it costs almost nothing. I would suggest having a check at `[1] === '.'` to be more careful. 😅

Reply:
Thought about it too :D